.github/workflows/lint_test_charts.yaml

```diff
@@ -141,3 +141,34 @@ jobs:
           cut -d\  -f2 versions.txt | sort -u
 
           [ $(cut -d\  -f2 versions.txt | sort -u | wc -l) -eq 1 ] || exit 1
+
+  bundle-dependencies-check:
+    name: Test the bundle dependency conditions
+    runs-on: ubuntu-latest
+    if: ${{ needs.lint-test.outputs.bundle-changed == 'true' }}
+    needs: [ lint-test ]
```

Comment:
It will be executed only when something is changed, but I tried it works by commenting this lines. Example: https://github.com/newrelic/helm-charts/actions/runs/3438721588/jobs/5735147287#step:6:1